package.json

```diff
@@ -35,7 +35,8 @@
     "webpack": "^2.6.1"
   },
   "dependencies": {
-    "bootstrap": "^4.1.1",
+    "@babel/runtime": "7.0.0-beta.55",
+    "bootstrap": "^3.3.6",
```

Comment:
Why is it necessary to go back here?

Reply:
Since cockpit uses redhat patternfly, which only supports bootstrap3